lua/lvim/config/supported_languages.lua

```diff
@@ -26,6 +26,7 @@ return {
   "fennel",
   "fish",
   "fortran",
+  "fsautocomplete",
```

Comment:
```suggestion
  "fsharp",
```

Reply:
No worries. I made the suggested change. I'm glad to hear I was not going crazy because I could not see how the `supported_languages.lua` are been used.